openapi.yml

```diff
@@ -359,15 +330,20 @@ components:
           $ref: '#/components/schemas/Provider'
         config:
           type: object
-        configValidation:
-          type: object
-          additionalProperties:
-            $ref: '#/components/schemas/ConfigValidation'
+        healthy:
+          type: boolean
+        diagnostics:
+          type: array
+          items:
+            $ref: '#/components/schemas/DiagnosticLog'
         schema:
           $ref: '#/components/schemas/ProviderSchema'
       required:
         - provider
         - config
+        - healthy
+        - diagnostics
+        - schema
```

Comment:
@JoshuaWilkes any reason why the `schema` field wasn't originally marked as required? I have now made it a required field.

Reply:
Just a mistake from what I can tell